tests/geometry/functional/test_so3.py

```diff
@@ -28,3 +28,13 @@ def test_adjoint(batch_size: int, dtype: torch.dtype):
     tangent_vector = torch.rand(batch_size, 3, dtype=dtype, generator=rng)
     group = so3.exp(tangent_vector)
     check_lie_group_function(so3, "adjoint", TEST_EPS, group)
+
+
+@pytest.mark.parametrize("batch_size", [1, 20, 100])
+@pytest.mark.parametrize("dtype", [torch.float32, torch.float64])
+def test_inverse(batch_size: int, dtype: torch.dtype):
+    rng = torch.Generator()
+    rng.manual_seed(0)
+    tangent_vector = torch.rand(batch_size, 3, dtype=dtype, generator=rng)
+    group = so3.exp(tangent_vector)
```

Comment:
would we rather generate a rand so3 or does that function also internally call exp?

Reply:
I'm working on this now. Should be pushed in the next two PRs.